script/QA/QA_19.txt

```diff
@@ -66,47 +71,49 @@ Why are you asking me for my opinion?
 [sha:42462f8f87e64703b159a283217a73469fbff21b]{
 -- Page 149, Offset 43592.
 -- えー、今回の原因は……
-Ehh, the cause of death this time...
+Hmm, the cause of death was...%{n}
 }
 [sha:cefaf839467c85996cb768471e7af8b46c5a2034]{
 -- Page 149, Offset 43593.
 -- 直前の選択肢ですね。
-must be from the previous choice.
+Ah, it was the previous choice.
 }
 [sha:ae6cf83927608cbd8665a0a0343216efb3da26ab]{
 -- Page 150, Offset 43594.
 -- 『戦う』という発想は良かったのですが、
-The thought of 'fighting' was definitely correct,
+It's good that you decided to fight, but%{n}
 }
 [sha:960979cdbf385432eab4a5524bb57d2bc3442a57]{
 -- Page 150, Offset 43595.
 -- 状況が最悪でした。
-but the situation was absolutely terrible.
+you picked the absolute worst time to do it.
 }
 [sha:8216b167fd4ec86060d1188d93abf8195eaa3fda]{
 -- Page 151, Offset 43596.
 -- 死徒になってもノエルは臆病なので、“その人間の
-Even as a Dead Apostle, Noel was a coward that
+// Is 一番重い所 supposed to be his soul or something?
+// That's what she ended up slurping out in the file
+Even after becoming a Dead Apostle, Noel was still a coward,%{n}
 }
 [sha:9acd877c53887bbaab8777295b3cb4d530ea17d8]{
 -- Page 151, Offset 43597.
 -- 一番重い所を利用する”事を避けていました。
-avoided going for the opponent's weakest point.
+so she'd been afraid of exploiting her opponent's soul.
```

Comment:
She means how Noel used his past to stop him in his track. Also, I think present tense for the first sentence is better. This is sort of taking place "in the present as seen in the last scene" so Noel is still alive, and still a vampire.

>Even as a Dead Apostle, Noel is still a coward, so she was avoiding handling her opponent's innermost darkness.

Something like that, I think get's the idea across. She's basically afraid to look too much at Shiki's traumas.